app/views/includes/forms/card.scala.html

```diff
@@ -19,26 +19,25 @@
   id: String,
   gaAction: Option[String] = None,
   heading: String,
-  headingTag: String = "h2",
+  headingTag: String = "h3",
   bodyContent: Option[String] = None,
   cardActionUrl: Option[String] = None,
   cardActionText: Option[String] = None
 )
 
 <div class="@cardClass" id="@id">
 
-  <div class="card-body @if(url){active}">
-
-    <@headingTag class="heading-small card-heading">
+  <div class="card-body cardSize @if(url){active}">
+    <@headingTag class="govuk-heading-s card-heading">
       @if(url){
-        <a class="card-link" href="@url">
+        <a class="card-linkxx" href="@url">
```

Comment:
is this meant to say linkxx?

Reply:
yes thnks  this is a typo - class has been removed